_raml/apps/scripttags/script-tags.raml

```diff
@@ -0,0 +1,34 @@
+#%RAML 0.8
+displayName: Script tags
+get:
+  description: Returns a list of all script tags for a shop. *epagesNow
+  queryParameters:
+    myScriptTagsOnly:
+      description: Shows script tags that have been created for a specific application only.
+      type: boolean
+      required : false
+  responses:
+    200:
+      body:
+        application/json:
+          example: !include examples/get_scripttags_response.json
+          schema: !include schema/get_scripttags.schema.json
+post:
+  description: Creates a new script tag. *epagesNow
+  body:
+    application/json:
+      example: !include examples/post_scripttag_request.json
+      schema: !include schema/post_scripttag_request.schema.json
+  responses:
+    201:
+      body:
+        application/json:
+          example: !include examples/post_scripttag_response.json
+          schema: !include schema/post_scripttag.schema.json
+
+/{scriptTagId}:
+  displayName: Script tag
+  delete:
+    description: Deletes a script tag identified by the provided script tag ID. *epagesNow
```

Comment:
Please mention also, that it is not possible to delete all the script tags of a shop, but only those that were created by the app itself.